pkg/metrics/collector/sockstat_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestSockstats(t *testing.T) {
+	input := "sockets: used 270\n" +
```

Comment:
const

Reply:
use the literal quote
```
const input = `sockets: used 270
TCP: inuse 23 orphan 0 tw 10 alloc 52 mem 6
...
`
```

Reply:
Change it to []string as the other comment suggested.